sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -113,28 +123,36 @@ export class ServiceBusClient {
   getReceiver(
     topicName: string,
     subscriptionName: string,
-    receiveMode: "receiveAndDelete"
+    receiveMode: "receiveAndDelete",
+    options?: GetReceiverOptions
   ): Receiver<ReceivedMessage>;
   getReceiver(
     queueOrTopicName1: string,
     receiveModeOrSubscriptionName2: "peekLock" | "receiveAndDelete" | string,
-    receiveMode3?: "peekLock" | "receiveAndDelete"
+    receiveModeOrOptions3?: "peekLock" | "receiveAndDelete" | GetReceiverOptions,
+    options4?: GetReceiverOptions
   ): Receiver<ReceivedMessageWithLock> | Receiver<ReceivedMessage> {
     let entityPath: string;
     let receiveMode: "peekLock" | "receiveAndDelete";
+    let options: GetReceiverOptions;
 
-    if (isReceiveMode(receiveMode3)) {
+    if (isReceiveMode(receiveModeOrOptions3)) {
```

Comment:
Now that #8074 is merged, there are changes to this place and you will need to merge from master

Reply:
Merged and resolved.